integr8ly/installation
# INTLY-2663 add user sso as msbroker env var

verification:
- run the installer against an openshift 3 cluster
- ensure the USER_SSO_URL env var is set against the msbroker
- ensure the deployed webapp version is 2.14.0
- ensure that when logging into the webapp
  - there are two visible SSO instances on the right hand side, one standard and one cluster
  - both links still work as expected
